ccf_specs.md

```diff
@@ -906,28 +974,30 @@ composite-type-value = [
 
 function-type-value =
     ; cbor-tag-function-type-value
-    #6.155([
-      id: tstr,
-      parameters: [
+    #6.193(function-untagged-type-value)
+
+function-untagged-type-value = [
+    id: tstr,
```

Comment:
What is this ID?

Reply:
As part of reply to prior comment, I changed this `id` to `cadence-type-id` in the same commit 13a5309d6e6cf683068470c68b0e9add21493185.